spec/features/login_page_spec.rb

```diff
@@ -0,0 +1,98 @@
+describe 'login_page' do
+  before { visit '/login' }
+
+  context 'Visibility' do
+    it 'has Email field' do
+      expect(page).to have_css('input#spree_user_email')
+    end
+
+    it 'has Password field' do
+      expect(page).to have_css('input#spree_user_password')
+    end
+
+    it 'has Remember_me checkbox' do
+      expect(page).to have_css('input#spree_user_remember_me')
+      expect(page).to have_css('label', text: 'Remember me')
+    end
+  end
+
+  context 'Login process' do
+    it 'can Log in as a User' do
+      fill_in 'spree_user_email', with: ENV['USERNAME_SPREE']
+      fill_in 'spree_user_password', with: ENV['PASSWORD_SPREE']
+      find_button('Login').click
+      expect(page).to have_css('a', text: 'Logout')
+      expect(find('.alert-success').text).to eql 'Logged in successfully'
+    end
+
+    it 'cannot Log in as a User with incorrect email' do
+      fill_in 'spree_user_email', with: '12345@qwe.co'
+      fill_in 'spree_user_password', with: '12345'
```

Comment:
I'd suggest using `let` for values which are re-used
For instance - you're using the same password in most of the specs
You can do something like
```
let(:password) { '12345' }
....
fill_in 'spree_user_password', with: password
```
This approach is good due to the fact that in case of changes you need to change value only in 1 place as opposed to every spec